### DISCUSSION TITLE
参考文献标点异常

### DISCUSSION THREAD

**Discussion by pyb0924** [sjtug/SJTUThesis#801] [提问] [answered]
![屏幕截图 2022-05-03 160035](https://user-images.githubusercontent.com/50937198/166421115-4bf35307-21b8-471a-b400-692159878160.png)

这个People’s好像用的是中文标点，但是参考文献里其他的标点符号都是正常的，想问一下这种情况要怎么解决。

**Comment by stone-zeng** [comment, marked as answer]
在 bib 里面把 `’` 改成 `'`（即中文蝌蚪引号改成西文直引号）
